contents/blog/why-companies-self-host.md

```diff
@@ -0,0 +1,80 @@
+
+Knowledge sharing tool [Saga](/customers/saga) ran a side-by-side comparison between Mixpanel and a self-hosted instance of PostHog and found a huge difference in the number of events that were tracked.
+
+"For one event we defined, Mixpanel reported 6,343 occurrences last month," explains Filip Stanev, co-founder of Saga. "PostHog reported what we know to be the true figure, 8,169 — almost 30% more than Mixpanel.”
+
+So what's going on here?
+
+Again, it's related to privacy, as our CTO and co-founder, Tim Glaser, explains:
+
+"Event tracking is typically prevented by ad blockers, which block tracking that originates from third-party domains. Self-hosting your analytics avoids this problem because the tracking originates from your domain. 
+
+"This behavior isn't limited to ad blockers, either. VPNs and privacy browsers, which aren't explicitly positioned as ad blockers, will generally block third-party trackers by default."
+
+Data quality is a critical issue for businesses who are now recognising the competitive advantage of data-driven product decisions. A [recent analysis](https://mikkeldengsoe.substack.com/p/data-to-engineers) of the top 50 European tech companies indicates high-growth startups are hiring data analysts at a higher rate than ever - banking app Monzo more than doubled the size of its data team in the last two years, while food delivery app Glovo has 74 open data roles compared to 102 open engineering roles.
+
+## 3) It's a principle
+
+Sometimes, a company's demands are philosophical rather than practical. We believe in transparency. That's why our product is open source and it's why everything about how we run the business, from [how we conduct meetings](/handbook/getting-started/meetings) or our [compensation calculator](/handbook/people/compensation), is available for everyone to see in our [company handbook](/handbook).
+
+PostHog customers frequently tell us that controlling their data through self-hosting is simply a core value of their business. They don't want to share their data with Google, Facebook or any other large third-party, both to protect their users but also the value of their proprietary data. For many this isn't a new desire, but until recently the option to self-host a powerful, fully-featured product analytics suite didn't exist.
+
+## We get it, self-hosting is great, does that mean everyone should do it?
+
+We are self-hosting evangelists and [deploying PostHog](/docs/self-host) is a quick and easy process, but we also recognise not everyone wants or is ready to self-host. That's why we [also offer PostHog Cloud](/signup), a version of PostHog hosted and managed by us with automatic upgrades, and minimal setup.
+
+We often find customers prefer to start on PostHog Cloud and migrate to self-hosting when they're ready. You can start using PostHog Cloud immediately and it's free to use up to 1 million events per month.
```

Comment:
2 cents: 
Maybe we can mention the benefit of PostHog cloud that you can see the source code, so even though it's SAAS it's still probably better in the eyes of privacy conscious folks.

'migrate to self-hosting when they're ready' / 'until you're ready to transition' leaves an impression that everyone should in the end self-host vs it could be better to leave the impression that it's possible and easy if they desire/need later.